marlowe-website/src/index.js

```diff
@@ -153,7 +154,51 @@ function fixCrossLinks() {
   }
 }
 
+function activateYoutube() {
+  $(".video-container iframe").each((_, elm) => {
+    elm.setAttribute("src", elm.getAttribute("data-src"));
+    elm.classList.remove("hidden");
+  });
+  $(".video-container p").each((_, elm) => {
+    elm.classList.add("hidden");
+    elm.classList.remove("flex");
+  });
+}
+function activateTypeform() {
+  var script = document.createElement("script");
+  script.type = "text/javascript";
+  script.async = !0;
+  script.src = "//embed.typeform.com/next/embed.js";
+  var node = document.getElementsByTagName("script")[0];
+  node.parentNode.insertBefore(script, node);
+}
+
+function initializeGDPR() {
+  function activateComponentsWithCookies() {
+    activateAnalytics();
+    activateYoutube();
+    activateTypeform();
+  }
+  const gdprConsent = localStorage.getItem("gdpr-consent");
+  if (gdprConsent === "agree") {
+    activateComponentsWithCookies();
+    $("#gdpr-banner").addClass("hidden");
+  } else {
+    // Show the banner
+    $("#gdpr-banner").removeClass("hidden");
+    $(".agreeGDPR").on("click", () => {
+      localStorage.setItem("gdpr-consent", "agree");
```

Comment:
Should we also track if they've declined, and prevent the banner from popping up each time they visit? I think that might be more courteous.

Reply:
Well, two thougts:
* If they don't allow us to store stuff in localstorage/cookies, can we store the rejection there?
* currently we show the message every time you access the page as is the only way to see the message again (no button or dialog for managing GDPR)